applications/nvidia_nim/nvidia_nim_chat/metadata.json

```
@@ -0,0 +1,36 @@
+{
+	"application": {
+		"name": "Chat with NVIDIA Inference Microservice (NIM)",
+		"authors": [
+			{
+				"name": "Holoscan Team",
+				"affiliation": "NVIDIA"
+			}
+		],
+		"language": "Python",
+		"version": "1.0",
+		"changelog": {
+			"1.0": "Initial Release"
+		},
+		"holoscan_sdk": {
+			"minimum_required_version": "1.0.3",
+			"tested_versions": [
+				"1.0.3"
+			]
```

The `./dev_container build` or `./dev_container build_and_run` options currently use v2.1.0 by default. Could you please do a quick test to verify that the app runs with v2.1.0 as well? Sufficient to confirm `./dev_container build_and_run nvidia_nim_chat` runs as expected.

```suggestion
			"tested_versions": [
				"1.0.3",
				"2.1.0"
			]
```

---

Alternatively, if you do want to pin to 1.0.3 you can add the` --base_img` flag to those `./dev_container` commands in the app README to use the 1.0.3 base container.

---

`./dev_container build_and_run nvidia_nim_chat` works as expected. `2.1.0` has been added to `tested_versions`.